sksparse/cholmod.pyx

```diff
@@ -145,6 +147,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
```

Comment:
I didn't mean necessarily for CholmodNotInstalledError, but e.g. for CholmodInvalidError (which should be triggerable?).  For example the error message of CholmodNotPositiveDefiniteError does already say the "L not positive definite", so it may not be necessary to wrap it in your own error message on top of it.

Also, it appears that the error needs to be decoded (ascii, I guess) into str on Py3.